sklearn/feature_selection/_sequential.py

```diff
@@ -30,11 +30,17 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin,
     estimator : estimator instance
         An unfitted estimator.
 
-    n_features_to_select : int or float, default=None
-        The number of features to select. If `None`, half of the features are
-        selected. If integer, the parameter is the absolute number of features
-        to select. If float between 0 and 1, it is the fraction of features to
-        select.
+    n_features_to_select : int or float, default='auto'
+        The number of features to select. If `'auto'`, features are selected
```

Comment:
We should be clearer here, with a bullet list:

```
If 'auto', the behaviour depends on the `tol` parameter:

- if `tol` is not None, then features are selected until the score improvement does not exceed `tol`.
- otherwise, half of the features are selected.
```

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/9b11fbbcfb1b443063a25940f4f8d3f5a4c6951e